src/utils/style-formatter.js

```diff
@@ -46,8 +51,9 @@ export default {
     styles += disabled ? formatProperty('opacity', 0.4) : formatProperty('cursor', 'pointer');
     // font
     styles += formatProperty('color', getColor(font, '#ffffff', theme));
-    font.style && font.style.bold && (styles += formatProperty('font-weight', 'bold'));
-    font.style && font.style.italic && (styles += formatProperty('font-style', 'italic'));
+    const fontStyle = font.style || DEFAULTS.FONT_STYLE;
```

Comment:
Where's `underline`?

Reply:
It is applied in `createLabelAndIcon`. Don't remember why, but it was something about the scaling of the label I think